packages/main/src/plugin/commands-init.ts

```diff
@@ -0,0 +1,134 @@
+/**********************************************************************
```

Comment:
I like the idea, and probably good for now, but I would argue that the commands should be registered by the class using it. 

E.g.

- `feedback` command, should probably be registered by `FeedbackHandler`
- `help` I don't know, kinda random
- `troubleshooting` by the navigation registry
- `pullImage` by the container registry... but the container registry is already a very big file... :/

I don't know, I would approve for now, as this is better that what we currently have, but might need to discuss architecture in the backend